.github/workflows/ci.yml

```diff
@@ -23,11 +23,15 @@ jobs:
     name: fuzz test
     runs-on: ubuntu-latest
     steps:
-      - uses: jidicula/go-fuzz-action@4f24eed45b25214f31a9fe035ca68ea2c88c6a13 # v1.2.0
+      # Pinned a commit to make go version configurable.
+      # This should be safe to upgrade once this commit is in a released version:
+      # https://github.com/jidicula/go-fuzz-action/commit/23cc553941669144159507e2cccdbb4afc5b3076
+      - uses: jidicula/go-fuzz-action@0206b61afc603b665297621fa5e691b1447a5e57
         with:
           packages: 'github.com/sourcegraph/zoekt' # This is the package where the Protobuf round trip tests are defined
           fuzz-time: 30s
           fuzz-minimize-time: 1m
+          go-version: '1.21'
```

Comment:
Needed to update this so the build doesn't fail when using new `go 1.21` features.